### DISCUSSION TITLE
How to install Losslesscut ver 3.58 to work on Debian ARM64

### DISCUSSION THREAD

**Discussion by kumback** [mifi/lossless-cut#1763] [Q&A]
I installed Losslesscut on Debian arm64 following the instruction below, and it works correctly, but it is ver3.44. How can I update to ver3.58 or how can I install ver3.58??

I tried the same way replacong 3.44 with 3.58, and it seems ver3.58 installed, but it does NOT work correctly, just black screen appears. Anyone knows how to have ver3.58 working on Debian arm64?? Any advice would be appreciated.

sudo apt update
sudo apt install libasound2 -y # this fixes libasound.so error in beta
sudo apt install wget ffmpeg -y
wget https://github.com/mifi/lossless-cut/releases/download/v3.44.0/LosslessCut-linux-arm64.tar.bz2
tar -xvf LosslessCut-linux-arm64.tar.bz2
cp /bin/ffmpeg ~/LosslessCut-linux-arm64/resources
cp /bin/ffprobe ~/LosslessCut-linux-arm64/resources
cd ~/LosslessCut-linux-arm64/
./losslesscut --no-sandbox

**Comment by mifi** [comment]
check to see if you're seeing any errors either in the console or developer tools inside electron(losslesscut ui)

**Comment by kumback** [comment]
Hello, thank you very much for your reply.

I have these errors which I do not have with ver3.44.

[3248:1101/225842.926926:ERROR:gbm_wrapper.cc(253)] Failed to export buffer to dma_buf: No such file or directory (2)
ffmpeg -hide_banner -f lavfi -i 'nullsrc=s=256x256:d=1' -f null -
[3248:1101/225844.183727:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 1 times!
[3248:1101/225844.190589:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 2 times!
[3248:1101/225846.860691:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 3 times!

**Comment by kumback** [comment]
and more like these.

Warning: loader_scanned_icd_add: Driver /usr/lib/aarch64-linux-gnu/libvulkan_lvp.so supports Vulkan 1.1, but only supports loader interface version 4. Interface version 5 or newer required to support this version of Vulkan (Policy #LDP_DRIVER_7)
Warning: loader_scanned_icd_add: Driver /usr/lib/aarch64-linux-gnu/libvulkan_radeon.so supports Vulkan 1.2, but only supports loader interface version 4. Interface version 5 or newer required to support this version of Vulkan (Policy #LDP_DRIVER_7)
Warning: Layer VK_LAYER_MESA_device_select uses API version 1.2 which is older than the application specified API version of 1.3. May cause issues.
Warning: eglChooseConfig returned zero configs
    at Create (../../third_party/dawn/src/dawn/native/opengl/ContextEGL.cpp:52)
    at Create (../../third_party/dawn/src/dawn/native/opengl/PhysicalDeviceGL.cpp:97)

**Comment by kumback** [comment]
![Screenshot 2023-11-02 07 21 32](https://github.com/mifi/lossless-cut/assets/61573275/ce4c8e41-1c73-45e9-8c1e-4097a3a328f2)

This kind of broken screen or black screen appears when start ver3.58.

**Reply by mifi** [reply]
wow that looks really bad. it's definitely a bug related to linux or electron, so there's not much I can do, other than encourage you to google the error messages that you received.

**Comment by kumback** [comment]
just i let you know, I installed v3.56 in the same way as v3.58. it has the same errors but unlike v3.58, no broken or black screen, and  it works as expected. i have no idea why????

![Screenshot 2023-11-02 23 57 13](https://github.com/mifi/lossless-cut/assets/61573275/2d8469a3-e430-4023-9345-19fcb4f6a0b3)

**Reply by mifi** [reply]
Electron version upgraded between those versions of llc

**Comment by kumback** [comment]
is there anything i should try??

**Reply by mifi** [reply]
i don't know other than degrade losslesscut or try to google the errors

**Reply by kumback** [reply]
Thank you very much. I keep using v3.56 which works in my environments.
